.travis.yml

```diff
@@ -22,27 +21,77 @@ before_cache:
 env:
   matrix:
     # Setting this variable twice will cause the 'script' section to run twice with the respective env var invoked
-    - BUILD_TYPE=centaurAws
-    - BUILD_TYPE=centaurBcs
-    - BUILD_TYPE=centaurEngineUpgradeLocal
-    - BUILD_TYPE=centaurEngineUpgradePapiV2
-    - BUILD_TYPE=centaurHoricromtalPapiV2
-    - BUILD_TYPE=centaurHoricromtalEngineUpgradePapiV2
-    - BUILD_TYPE=centaurPapiUpgradePapiV1
-    - BUILD_TYPE=centaurPapiUpgradeNewWorkflowsPapiV1
-    - BUILD_TYPE=centaurLocal
-    - BUILD_TYPE=centaurPapiV1
-    - BUILD_TYPE=centaurPapiV2
-    - BUILD_TYPE=centaurSlurm
-    - BUILD_TYPE=centaurTes
-    - BUILD_TYPE=centaurWdlUpgradeLocal
-    - BUILD_TYPE=checkPublish
-    - BUILD_TYPE=conformanceLocal
-    - BUILD_TYPE=conformancePapiV2
-    - BUILD_TYPE=conformanceTesk
-    - BUILD_TYPE=dockerDeadlock
-    - BUILD_TYPE=dockerScripts
-    - BUILD_TYPE=sbt
+    - >-
+      BUILD_TYPE=centaurAws
+      BUILD_MYSQL=5.7
```

Comment:
The MySQL we use in production is 5.6.36, probably doesn't matter much if at all

Reply:
Good to know. We can maybe downgrade if we'd like in the future.